assets/js/blocks/cart-checkout/cart/checkout-button/style.scss

```diff
@@ -6,3 +14,43 @@
 		margin-bottom: 0;
 	}
 }
+
+.is-mobile,
+.is-small,
+.is-medium {
+	.wc-block-cart__submit-container:not(.wc-block-cart__submit-container--sticky) {
+		padding-left: 0;
+		padding-right: 0;
+		padding-top: 0;
+	}
+}
+
+@include breakpoint( ">782px" ) {
+	.wc-block-cart__submit-container--sticky {
+		display: none;
+	}
+}
```

Comment:
I'm not sure to understand what you mean. The CSS breakpoint is there to make the sticky container to be displayed only on small viewports.

Reply:
What I mean is that is that it looks like the `.wc-block-cart__submit-container--sticky` will only be rendered when the `positionRelativetoViewport === 'below'` (as found in [this code block](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/828d4dd3c5b86c5117be875efe3173de0620cd8a/assets/js/blocks/cart-checkout/cart/checkout-button/index.js#L66-L70)). So there's no need for a display property? 

However, looking the code again, it looks like the IntersectionObserver `positionRelativeToViewport` is going to be set regardless of viewport width and thus the need to add `display:none;` here because the sticky container will always be present given the conditions across various viewport widths. I think what tripped me up when doing the review was an [earlier response you gave](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/3004#discussion_r473054340) and I was conflating that in my mind that the intersection observation would only be happening in mobile views:

> I don't think so, the sticky behavior is only available on mobile so no need to add support for IE11.

Given the new perspective of this, I think I understand why the css is needed now.